app/app.go

```diff
@@ -879,8 +903,8 @@ func (app *StrideApp) BlacklistedModuleAccountAddrs() map[string]bool {
 	modAccAddrs := make(map[string]bool)
 	// DO NOT REMOVE: StringMapKeys fixes non-deterministic map iteration
 	for _, acc := range utils.StringMapKeys(maccPerms) {
-		// don't blacklist stakeibc module account, so that it can ibc transfer tokens
-		if acc == "stakeibc" {
+		// don't blacklist stakeibc, alliance module account, so that it can ibc transfer tokens
+		if acc == "stakeibc" || acc == alliancemoduletypes.ModuleName {
```

Comment:
why does alliance need to ibc transfer tokens?

Reply:
I think we can actually get rid of this code, this was here due to an IBC bug, but now the bug is patched.

Reply:
> I think we can actually get rid of this code, this was here due to an IBC bug, but now the bug is patched.

I don't think blacklisted addresses are only for ibc transfer because blacklisting is used to block bank transfer to blocklisted addresses.
Its necessary to exclude the alliance module address from blacklisted addresses because alliance asset rewards are claimed from distribution module to alliance module.

Reply:
Got it